packages/aws-cdk-lib/core/lib/cfn-fn.ts

```diff
@@ -500,9 +502,31 @@ class FnFindInMap extends FnBase {
    * @param mapName The logical name of a mapping declared in the Mappings section that contains the keys and values.
    * @param topLevelKey The top-level key name. Its value is a list of key-value pairs.
    * @param secondLevelKey The second-level key name, which is set to one of the keys from the list assigned to TopLevelKey.
+   * @param defaultValue The value of the default value returned if either the key is not found in the map
```

Comment:
Does this need to indicate that the parameter is optional in some way?

Reply:
no i dont think so